superset-frontend/src/utils/urlUtils.ts

```diff
@@ -134,14 +133,7 @@ function getPermalink(endpoint: string, jsonPayload: JsonObject) {
   return SupersetClient.post({
     endpoint,
     jsonPayload,
-  })
-    .then(result => result.json.url as string)
-    .catch(response =>
-      // @ts-ignore
-      getClientErrorObject(response).then(({ error, statusText }) =>
-        Promise.reject(error || statusText),
-      ),
-    );
```

Comment:
we'll never get an error here?

Reply:
Errors should be handled downstream---when it's about to be rendered. These adhoc handling of API errors makes it more difficult to handle errors consistently.